scripts/deploy-rpc-playbooks.sh

```diff
@@ -29,3 +28,9 @@ if [[ "${DEPLOY_ELK}" == "yes" ]]; then
     run_ansible haproxy.yml
   fi
 fi
+
+#verify RAXA MaaS is running after all necessary
```

Comment:
Nit: All comments should have spaces after the octothorpe. Also what is "RAXA" ;)